### PR TITLE
Gravity Radio Update of metadata

### PR DESCRIPTION
**Pull request by jonathanjamandsimi**

### CODE REVIEW

**Comment by Bas950** [comment]
We need screenshot proof that your new presence is working.

**Comment by jonathanjamandsimi** [comment]
Screenshot_1

**Comment by jonathanjamandsimi** [comment]
Thats the picture.

File: Uptimerobot/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : new P(function (resolve) { resolve(result.value); }).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
```
Comment (Timeraa) [inline Uptimerobot/dist/presence.js]: Why are people adding stuff from the TypeScript compiler?

**Pull request merged**